pkg/controller/commonwebuiservice/commonwebuiservice_controller.go

```diff
@@ -918,3 +918,40 @@ func (r *ReconcileCommonWebUI) createRedisCustomResource(unstruct unstructured.U
 	}
 	return nil
 }
+
+func (r *ReconcileCommonWebUI) finalizerRedisSentinelCr(instance *operatorsv1alpha1.CommonWebUI, unstruct unstructured.Unstructured) {
+	reqLogger := log.WithValues("Instance.Namespace", instance.Namespace, "Instance.Name", instance.Name)
+
+	finalizerName := "commonui1.operators.ibm.com"
```

Comment:
@Scocconcelli Please reuse the `finalizerCr` function that we have to finalize the redis sentinel cr. I see that the only change is in the `finalizerName` so you can pass it as a parameter to the  already present `finalizerCr` function and reuse it.